Chapter05/Lesson21/userEdit/app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,27 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center"
+    android:orientation="vertical"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    tools:context=".MainActivity">
+
+    <!--通知を表示するServiceを起動するButton-->
+    <Button
+        android:id="@+id/buttonShow"
```

Comment:
Generally, we seem to use snake case in case of `android:id`. Snake case is like a `foo_bar`.
http://d.hatena.ne.jp/glass-_-onion/20080512/1210580746